newjersey/navigator.business.nj.gov
# chore: [#186335653] remove search-business-name, searchOnly

## Description



This pull request resolves Pivotal Tracker ticket #186335653 by removing the legacy `search-business-name` task and the `searchOnly` prop from `<BusinessFormation />` and `<BusinessFormationPaginator />`. Additionally, it contains code changes that clear the current set of frontend feature flags (as all feature flag-hidden work is now deployed) and removes any tests that depended on the feature flags to be set to `false`.

### Ticket



[#186335653](https://www.pivotaltracker.com/story/show/186335653)

### Approach



This pull request does not require any work on the user's part to enable functionality. It's generally a removal of legacy, un-needed code.

However, once this is merged, you can remove the relevant feature flags in your `web/.env` file.

### Steps to Test



Automated tests were run using `./scripts/pre-commit.sh`. There are no user-facing changes other than the user should no longer see the legacy `search-business-name` task anymore.

### Notes



No additional notes are provided here.

## Code author checklist
